src/actions/__tests__/importText.ts

```diff
@@ -707,7 +707,6 @@ describe('HTML content', () => {
   })
 
   it('should paste text properly that is copied from OSX Notes.app', () => {
-    /* eslint-disable no-irregular-whitespace */
```

Comment:
?

Reply:
Basically as per eslint v9.

if there is no rule define in eslint.config.ts and we are adding the command in our code. so eslint lint will show that command as warning.

so we just fix all these by running yarn lint --fix. if you want to keep these warning as it is so will revert all these.